packages/mdx-components/src/main/card-group/card-group.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid } from '@carbon/react'
+import { clsx } from 'clsx'
+import PropTypes from 'prop-types'
+import React, { ReactNode } from 'react'
+
+// TODO: ask Joe - how can we use this in here?
+// import { mediaQueries, useMatchMedia }
+// from '../../../../services/web-app/utils/use-match-media.js'
```

Comment:
@jdharvey-ibm tagging you now so I don't forget to ask you next week

Reply:
@alisonjoseph There's no easy way for us to use this in both places :/

I think core Carbon does have a media query hook that we could get at. That could potentially be used here instead.

If not, our best option is to copy-paste those two imports from the web-app into the mdx-components package.